src/fireflyClient.ts

```diff
@@ -258,11 +258,10 @@ export class FireflyClient {
   /***
    * Set UUID to api headers for colocation partners
    */
-  setUUID = (uuid:string) => {
-       if(uuid!="")
-       {
-        this.apiService.setUUID(uuid);
-       }
+  setUUID = (uuid: string) => {
+    if (uuid != "") {
+      this.apiService.setUUID(uuid);
+    }
```

Comment:
Same thing as we discussed in the python client, we should allow users to set the UUID to `""`

Reply:
Wait nvm, I see this in the other commit, thanks for changing it!